openquake/hazardlib/gsim/can15/oceanic.py

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright (C) 2014-2018 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""
+:module:`openquake.hazardlib.gsim.can15.western` implements
+:class:`OceanicCan15Mid`, :class:`OceanicCan15Low`, :class:`OceanicCan15Upp`
+"""
+
+import numpy as np
+
+from openquake.hazardlib.gsim.can15.western import WesternCan15Mid
+from openquake.hazardlib.gsim.can15.western import get_sigma
+
+
+class OceanicCan15Mid(WesternCan15Mid):
+    """
+    Implements the GMPE for oceanic sources
+    """
+
+    def get_mean_and_stddevs(self, sites, rup, dists, imt, stddev_types):
+        """ """
```

Comment:
If a docstring is not interesting, it is better to remove it than to have it empty

Reply:
Fixed